README.md

```diff
@@ -32,8 +38,21 @@ test/
 * [Mocha](https://mochajs.org)
 * [Chai](http://chaijs.com)
 
-
+  
 ### Useful Resources
 * [Thinking in React](https://facebook.github.io/react/docs/thinking-in-react.html)
 * [When to make a something part of the component state](https://twitter.com/dan_abramov/status/749710501916139520)
 * [Containers vs. Components](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0#.83k2l937e)
+
+### Style Guide
```

Comment:
Can we move the Style Guide and Contributing Guidelines sections above the Tools/Plugins and Useful Resources sections, since they are more relevant to someone looking at the README.md?